src/plugins/vis_type_tagcloud/public/components/tag_cloud_visualization.test.js

```diff
@@ -0,0 +1,176 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import 'jest-canvas-mock';
+
+import { ExprVis } from '../../../visualizations/public/expressions/vis';
```

Comment:
Please import this from the top level of `visualizations/public`. It should be called `ExprVisClass` there: https://github.com/flash1293/kibana/blob/19bda1fceeca564d5bae73b3a78276870e25f220/src/plugins/visualizations/public/index.ts#L24

Reply:
There's no export of`ExprVisClass` in `visualizations/public`, and `ExprVis` is exported like type, so it's constructor can't be accessed in tests. Here's related code from index.ts:
`import { ExprVis as ExprVisClass } from './expressions/vis';`
`export type ExprVis = ExprVisClass;`

@ppisljar I am wondering why do we need here export like this and is there's an opportunity to replace it with 
`export { ExprVis } from './expressions/vis';`

Reply:
Ah, good point, I missed this. In this case we should inline the part of it we need (like here: https://github.com/flash1293/kibana/blob/bf04235dae35452061cc7ea3d86d96c19a58206c/src/plugins/vis_type_metric/public/components/metric_vis_component.test.tsx#L44 )